tests/BUILD

```diff
@@ -0,0 +1,8 @@
+load("//proto:defs.bzl", "proto_library")
+
+proto_library(
```

Comment:
What is this rule for? It doesn't seem to be used.

Reply:
It's used as a (very very simple) test.